ui/src/views/Documentation/Reference/Entry.jsx

```diff
@@ -101,7 +102,9 @@ export default class Entry extends Component {
   state = {
     expanded:
       this.props.entry.name === window.location.hash.slice(1) ||
-      this.props.entry.type === window.location.hash.slice(1),
+      this.props.entry.type === window.location.hash.slice(1) ||
+      encodeURIComponent(get(this.props.entry, 'content.$id')) ===
```

Comment:
`get` is a lodash method. Let's either use `ramda` or plain javascript.

Reply:
Plain javascript is crashing for `api` and `exchange` pages, so we would need something which can handle exceptions, I will change it to `ramda`

Reply:
@helfi92 I can't find `get` in `ramda` docs, is it names something else? I went through the document, couldn't find any related to get

Reply:
It's not necessarily named `get`. Perhaps https://ramdajs.com/docs/#prop?